### DISCUSSION TITLE
Recommended stacks folder structure base is /user/home/docker

### DISCUSSION THREAD

**Discussion by mgrimace** [louislam/dockge#490] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I'm setting up multiple services inside 2-3 large compose files to make use of things like anchors, etc. Each app's config lives in /home/user/docker/appdata/[app name].

Presently, I have only one stack at /home/user/docker/compose.yaml. I plan to split that into separate media and networking stacks, and maybe a few others. 

What would be a logical place to put all the composes to work with dockge?
For example /home/user/docker/**stacks**/media/compose.yaml vs., /home/user/docker/media/compose.yaml? 

Any advice?

### 🐻 Dockge Version

Current

### 💻 Operating System and Arch

Debian 12

### 🌐 Browser

Safari

### 🐋 Docker Version

Cuurent

**Comment by trythatagain** [comment, marked as answer]
TBH not sure it makes much difference, what ever works for you, I have a compose folder for my compose/stacks, so for me, I'd choose the 1st, especially as you also have a appdata folder.

**Reply by mgrimace** [reply]
Thanks so much for the advice! In your case, how do you handle the .env(s)? For example, my 'base' .env has core info that's used across compose/stacks like PUID, GUID, TZ, etc, etc. how do you avoid repeating the same .env per folder
?

**Comment by trythatagain** [comment]
At the moment there is no global .env, but I believe it has been worked on and is just waiting for the next version. I did see that [ [Feature Request] Global Environmental Variables #133 ](https://github.com/louislam/dockge/discussions/133l) does show to do generate a updated image, but I'll be waiting for the official release. 
So for now you'll have to use the local .env file untill this comes out.
